src/core_plugins/state_session_storage_redirect/index.js

```diff
@@ -6,7 +6,7 @@ export default function (kibana) {
         title: 'Redirecting',
         id: 'stateSessionStorageRedirect',
         main: 'plugins/state_session_storage_redirect',
-        listed: false,
+        hidden: true,
```

Comment:
I'm not sure if `hidden` is the right word here.  To me, if something is hidden, it is inaccessible, but in this case it just means the app does not appear in the sidebar, correct?  I can live with it, it just feels off.

Reply:
Actually `hidden` means it doesn’t show in the navigation and isn’t accessible via /app/{id}. `listed` means that the app isn’t rendered in the nav but is accessible via the app route

Reply:
Ah, that makes sense.  Thanks for clarifying.